spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestUpdate.java

```diff
@@ -662,17 +665,6 @@ public void testUpdateWithNotInSubquery() {
         sql("SELECT * FROM %s ORDER BY id ASC NULLS LAST, dep", tableName));
   }
 
-  @Test
-  public void testUpdateWithNotInSubqueryNotSupported() {
```

Comment:
Looks like this is no longer needed because `testUpdateWithNotInSubquery` is not ignored anymore.

Reply:
Exactly. It is a valid test now.